tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Checkboxes', () => {
+    test.beforeEach( async({page}) => {
+        await page.goto('/')
+        //1-2. Opening dropdown list and select 'All' option then validating 'Veterinarians' page is opened
+        await page.getByRole('button', {name: "Veterinarians"}).click()
+        await page.getByRole('link', {name: 'All'}).click()
+        await expect(page.locator('h2')).toHaveText('Veterinarians')
+    })
+
+    test('Test Case 1: Validate selected specialties', async ({page}) => {
+        //3. Locating targeted row and clicking 'Edit'
+        const tableRows = page.getByRole('row', { name: 'Helen Leary' })
+        await tableRows.getByRole('button', {name: "Edit Vet"}).click()
+        //4. Asserting specialties fields has value 'radiology'
+        const specialtiesField = page.locator('.dropdown .selected-specialties')
+        await expect(specialtiesField).toHaveText('radiology')
+        //5. Clicking specialties field to show up menu with checkboxes
+        await specialtiesField.click()
+        //6. Asserting 'radiology' is checked
+        const checkBox1 = page.getByRole('checkbox', {name:'radiology'})
```

Comment:
This is a very bad naming pattern like 1, 2, 3 and so on. Always provide a good meaningful naming for the constants, variables, and method names.